chapter06/mssw/file_reader.pm

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/perl
```

Comment:
You don't need the shell to know how to execute this - it's only loaded by other perl scripts. Probably worth giving this a namespace (`package Mssw::CoconetReport;`) and an `@EXPORT_OK` list, so that it behaves as a typical perl module.